applications/GeoMechanicsApplication/custom_workflows/dgeoflow.cpp

```diff
@@ -216,33 +214,31 @@ namespace Kratos
         std::size_t found = name.find_last_of('.');
         std::string subname = name.substr(found + 1);
         ModelPart &part = model_part.GetSubModelPart(subname);
-        processes.push_back(make_shared<ApplyConstantScalarValueProcess>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_X,
+        mProcesses.push_back(make_shared<ApplyConstantScalarValueProcess>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_X,
                                                                                                          0.0, 0, ApplyConstantScalarValueProcess::VARIABLE_IS_FIXED)));
 
-        processes.push_back(make_shared<ApplyConstantScalarValueProcess>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_Y, -9.81,
+        mProcesses.push_back(make_shared<ApplyConstantScalarValueProcess>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_Y, -9.81,
                                                                                                          0, ApplyConstantScalarValueProcess::VARIABLE_IS_FIXED)));
 
-        processes.push_back(make_shared<Process>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_Z, 0.0,
+        mProcesses.push_back(make_shared<Process>(ApplyConstantScalarValueProcess(part, VOLUME_ACCELERATION_Z, 0.0,
                                                                                  0, ApplyConstantScalarValueProcess::VARIABLE_IS_FIXED)));
```

Comment:
And also here (note the template parameter now must be updated too):
```suggestion
        mProcesses.push_back(make_shared<ApplyConstantScalarValueProcess>(part, VOLUME_ACCELERATION_Z, 0.0,
                                                                                 0, ApplyConstantScalarValueProcess::VARIABLE_IS_FIXED));
```